300-CNAB-security.md

```diff
@@ -5,168 +5,85 @@ weight: 300
+While SSL / TLS protects users from man-in-the-middle (MitM) attacks, it is _not_ compromise-resilient, because attackers who compromise the infrastructure can abuse the online SSL / TLS key to replace files signed in transit but not at rest. Likewise, other solutions, such as GPG or X.509, do not support necessary features such as in-band key revocation or rotation.
 
-### Composing Layouts
+We propose using [The Update Framework (TUF)](https://theupdateframework.com) and [in-toto](https://in-toto.io) to solve this problem. TUF is a framework that provides security between _registries_, or the servers used to distribute bundles or images, and end-users. Specifically, TUF ensures the authenticity and integrity of bundles and images downloaded from registries. in-toto is a framework that provides security between developers and registries. Specifically, in-toto ensures the end-to-end integrity of the _software supply chain_ used to build and sign bundles or images. By combining both, we get a system that provides compromise-resilience between developers and end-users.
 
-Layouts are well-defined in the In-Toto specification. However, a certain subset of In-Toto features are considered OPTIONAL in the present spec:
+ Signing and verifying [claims](400-claims.md) are out of the scope of this document.
 
-- Inspections: In-Toto provides a method for running arbitrary commands on a machine in order to verify that specific parts of a software supply chain are correct. This specification does not require support for inspections. Moreover, this specification discourages allowing bundles to run external commands even for inspection purposes.
+## Roadmap
 
-### Verifying Layouts
+The basic idea is to TUF as a transport protocol that distributes several files in a compromise-resilient manner:
 
-The In-Toto specification defines the procedure for verifying individual links against the layout. This section adds CNAB-specific context to the more general specification.
+* The root of trust for all bundles and images, as well as TUF and in-toto metadata.
+* The software supply chains defined using in-toto.
+* The public keys used to verify these supply chains.
 
-If any part of a layout cannot be satisfied, the client SHOULD generate an error and MUST NOT continue installing the bundle.
+While images and bundles are published on registries, TUF and in-toto metadata about images and bundles are published on [_metadata repositories_](301-CNAB-metadata-repositories.md). Metadata repositories are not conceptually, but not necessarily physically, distinct from [CNAB](200-CNAB-registries.md) or [OCI](https://github.com/opencontainers/distribution-spec/blob/master/spec.md) registries. Furthermore, the key management models on metadata repositories are largely prescriptive. For these reasons and more, metadata repositories out of the scope of this document.
```

Comment:
Should the second sentence read, "Metadata repositories are conceptually..."? (Dropping the `not`?)

Reply:
Should the last sentence read, "For these reasons and more, metadata repositories _are_ out of the scope of this document."? (Missing `are`?)